llama/credit_card_slips.py

```diff
@@ -93,6 +96,18 @@ def get_po_title(po_line_record):
     return title
 
 
+def get_total_price(po_line_record, unit_price):
+    """Get total price from fund distribution of PO line record. If no  amount is listed
+    in the fund distribution, the unit price is returned as the total price."""
+    total_price = 0
+    for fund in [f for f in po_line_record["fund_distribution"] if f["amount"]["sum"]]:
```

Comment:
This will fail with a `KeyError` if either "amount" or "sum" keys are not present. `f.get("amount", {}).get("sum")` will return `None` if either is not present and this will work as expected.

Similarly, what if there's no `po_line_record["fund_distribution"]` field? The Alma object model isn't clear on which (if any) of these fields are actually required, so we should probably anticipate the possibility that they may not be present, even if we're fairly sure they will be...